recipes/aicssegmentation/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "aicssegmentation" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aicssegmentation-{{ version }}.tar.gz
+  sha256: 65a6e695ef0c5d4afe224e31259ae6cb8144aa474952a13c99d7426b2cc3f87b
+
+build:
+  entry_points:
+    - batch_processing=aicssegmentation.bin.batch_processing:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pytest-runner >=5.2
+    - python >=3.7
+  run:
+    - aicsimageio >=4.0.5
+    - dask-core
+    - itk
+    - itk-meshtopolydata
+    - itkwidgets
+    - jupyter
+    - matplotlib-base
+    - numpy >=1.15.1
+    - pandas >=0.23.4
+    - python >=3.7
+    - scikit-image >=0.18.0
+    - scipy >=1.1.0
+
+test:
+  imports:
+    - aicssegmentation
+  commands:
+    - pip list
+    # - pip check
+    - batch_processing --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/AllenCell/aicssegmentation
+  summary: Part 1 of Allen Cell and Structure Segmenter
+  license: BSD-2-Clause
+  license_file: LICENSE
```

Comment:
@jxchen01 , can you clarify this BSD license derivative? The 3rd point, in particular. Is conda-forge authorized to redistribute it or not?

> 
>    
> Allen Institute Software License – This software license is the 2-clause BSD
> license plus a third clause that prohibits redistribution and use for
> commercial purposes without further permission.
> 
> Copyright © 2020
> Jianxu Chen, Allen Institute.  All rights reserved.
> 
> Redistribution and use in source and binary forms, with or without
> modification, are permitted provided that the following conditions are met:
> 
> 1. Redistributions of source code must retain the above copyright notice, this
> list of conditions and the following disclaimer.
> 
> 2. Redistributions in binary form must reproduce the above copyright notice,
> this list of conditions and the following disclaimer in the documentation
> and/or other materials provided with the distribution.
> 
> 3. Redistributions and use for commercial purposes are not permitted without
> the Allen Institute’s written permission. For purposes of this license,
> commercial purposes are the incorporation of the Allen Institute's software
> into anything for which you will charge fees or other compensation or use of
> the software to perform a commercial service for a third party. Contact
> terms@alleninstitute.org for commercial licensing opportunities.
> 
> THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
> ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
> WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
> DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
> FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
> DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
> SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
> CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
> OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
> OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

Reply:
Pinging @jxchen01 :)

Reply:
sorry for the late reply. Our program manager is working on this and will help to explain more about the license shortly @[Kimcmetzler](https://github.com/Kimcmetzler)

Reply:
@jaimergp For point #3- It is fine to redistribute as long as the redistribution and does not incorporate the Allen Institute's software into a package whereby you will charge fees or sell the software for profit.

Reply:
Merged but we need to create a new license identifier for this derivative. Please ping me once the feedstock has been created!

@goanpeca @jxchen01